### DISCUSSION TITLE
I made a VS Code extension for Typst math calculating based on sympy :-)

### DISCUSSION THREAD

**Discussion by OrangeX4** [typst/typst#1371] [Show and tell]
VS Code extension for Typst math calculating, includes Arithmetic, Calculous, Matrix, Custom Variances and Funcions by yourself.

**Repo link:** https://github.com/OrangeX4/vscode-typst-sympy-calculator

![Demo](https://picgo-1258602555.cos.ap-nanjing.myqcloud.com/typst-sympy-calculator.gif)

**Comment by laurmaedje** [comment]
Really cool, you should also post this in the showcase on Discord!

**Reply by OrangeX4** [reply]
Thank you for liking it! But I don't have a Discord account yet, so you can post this GIF to Discord if you want. I'll try to sign up for an account later. Thank you also for creating typst, it's very great! :-)

**Reply by laurmaedje** [reply]
If you are going to create an account later anyway, I'll leave it to you, so that people can discuss it with the actual creator. :)
